src/OrleansTestKit/Storage/StorageManager.cs

```diff
@@ -1,32 +1,57 @@
 ﻿using System;
+using System.Collections.Generic;
+using Moq;
 using Orleans.Core;
+using Orleans.Runtime;
 
 namespace Orleans.TestKit.Storage
 {
     public sealed class StorageManager
     {
         private readonly TestKitOptions _options;
 
-        private object _storage;
+        private readonly Dictionary<string, object> _storages = new Dictionary<string, object>();
 
         public StorageManager(TestKitOptions options) =>
             _options = options ?? throw new ArgumentNullException(nameof(options));
 
-        public IStorage<TState> GetStorage<TState>()
+        internal readonly Mock<IAttributeToFactoryMapper<PersistentStateAttribute>> stateAttributeFactoryMapperMock =
+            new Mock<IAttributeToFactoryMapper<PersistentStateAttribute>>();
```

Comment:
This interface looks easy to implement without using a mock. Let's remove the mock.